core/src/main/java/de/jplag/merging/Neighbor.java

```diff
@@ -0,0 +1,9 @@
+package de.jplag.merging;
+
+import de.jplag.Match;
+
+/*
+ * This class realizes a pair of neighboring matches, named upperMatch and lowerMatch
```

Comment:
Could you add your definition of neighbors (both sides next to each other, without interrupting matches)? Remember in our meeting when you asked me for four matches what are the neighbors? To avoid ambiguity, just explain it here.